### PR TITLE
Fix/3134 old popup is appearing ... 

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
This PR adds the downloading and detecting states to exposure detection view.

Story: [3134](https://jira.itc.sap.com/browse/EXPOSUREAPP-3134)

IMG_0005
IMG_0006

### CODE REVIEW

**Review by marcussc — APPROVED** [review]
Good Job :)
Gotta say I like the new PR-Naming and having Screenshots in the PR 👍

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -138,6 +138,14 @@
 
 "ExposureDetection_Button_Refresh" = "Aktualisieren";
 
+"ExposureDetection_Risk_Status_Downloading_Title" = "Daten werden heruntergeladen …";
+
+"ExposureDetection_Risk_Status_Detecting_Title" = "Prüfung läuft …";
+
+"ExposureDetection_Risk_Status_Downloading_Body" = "Das kann bis zu mehrere Minuten dauern. Vielen Dank für Ihre Geduld.";
```
Comment (SabineLoss) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:145]:
```suggestion
"ExposureDetection_Risk_Status_Downloading_Body" = "Das kann mehrere Minuten dauern. Vielen Dank für Ihre Geduld.";
```

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -138,6 +138,14 @@
...
+
+"ExposureDetection_Risk_Status_Detecting_Body" = "Ihre Begegnungsdaten werden nun geprüft. Das kann bis zu mehrere Minuten dauern. Vielen Dank für Ihre Geduld.";
```
Comment (SabineLoss) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:147]:
```suggestion
"ExposureDetection_Risk_Status_Detecting_Body" = "Ihre Begegnungsdaten werden nun geprüft. Das kann mehrere Minuten dauern. Vielen Dank für Ihre Geduld.";
```

**Review by SabineLoss — COMMENTED** [review]
UA review - the changes in German are only related to German grammar and don't affect the other languages.

**Review by SabineLoss — APPROVED** [review]
UA reviewed

**Pull request merged**